admin/src/templates/edit_user.html

```diff
@@ -104,15 +104,38 @@
       <div class="help-block" translate>configuration.enable.token.login.refresh.help</div>
     </div>
 
-    <div class="form-group" ng-class="{ 'has-error': errors.password, 'required': !editUserModel.id, 'hidden': allowTokenLogin && (editUserModel.token_login || (editUserModel.token_login !== false && editUserModel.tokenLoginEnabled)) }">
+   <div 
+      class="form-group" 
+      ng-class="{ 'has-error': errors.password, 'required': !editUserModel.id, 'hidden': allowTokenLogin && (editUserModel.token_login || (editUserModel.token_login !== false && editUserModel.tokenLoginEnabled)) }">
       <label for="edit-password" translate>Password</label>
-      <input id="edit-password" name="edit-password" type="password" class="form-control" ng-model="editUserModel.password" autocomplete="new-password" ng-disabled="editUserModel.role === '_admin'"/>
+      <div class="password-input-group">
+        <input 
+          id="edit-password" 
+          name="edit-password" 
+          type="{{editUserModel.passwordFieldType}}"
+          autocomplete="new-password"
```

Comment:
```suggestion
```

I see now, thanks for explaining. Sonar is indeed complaining because it cannot see the hard-coded value 'password'

You can remove it